mantidimaging/gui/windows/operations/model.py

```diff
@@ -128,6 +132,18 @@ def apply_to_images(self, images: ImageStack, progress=None) -> None:
         exec_func = self.selected_filter.execute_wrapper(**input_kwarg_widgets)
         exec_func.keywords["progress"] = progress
         exec_func(images)
+
+        if is_preview:
+            return
+
+        params = ', '.join(f"{k}={v!r}" for k, v in exec_func.keywords.items() if k != "progress")
+        start = datetime.now()
+        LOG.info(f"Starting operation: {self.selected_filter.__name__} "
+                 f"(shape={images.data.shape}, {params})")
+
+        duration = (datetime.now() - start).total_seconds()
+        perf_logger.info(f"{self.selected_filter.filter_name} completed in {duration:.3f}s")
```

Comment:
Can these be in `if not is_preview:`. Otherwise this is changing the behaviour of when `record_operation` is called because the function returns early for `if is_preview:`

Also, the `start = ...` needs to be moved before `exec_func()`